### DISCUSSION TITLE
serverURL not reflecting with dev

### DISCUSSION THREAD

**Discussion by ralphilius** [payloadcms/payload#1828] [Q&A]
# Bug Report

## Steps to Reproduce

1. Set up payload with Github Codespaces
2. Set up serverURL with the preview URL of the Codespaces
3. Access the admin panel

## Other Details
 - Payload version 1.3.4
 - I've debugged through Browser Console, the `fetchMe` method seems to fetch from `localhost:3000`, not the server URL I specified.

**Comment by jmikrut** [comment]
Are you starting in Codespaces with `yarn dev` or similar?  Note that if you are starting with `yarn serve` and you update an env variable / similar, the admin UI will use its pre-built version and not reflect your changes. You need to run in dev mode, or rebuild, in order to see admin UI changes.

This is a problem with your implementation and not Payload itself, so I'll convert this to a discussion and we can help you out there. 

Can you share your repo?

**Reply by ralphilius** [reply]
I use `yarn dev` command
